src/SFA.DAS.EmployerFinance.Web/nlog.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
```

Comment:
Can `nlog.xsd` be included in the project to prevent the auto completion errors when viewing this file in an IDE?

Reply:
Added Nuget for Nlog Schema which should contain this now